lighthouse-core/config/config.js

```diff
@@ -62,7 +62,8 @@ function validateCategories(categories, audits, groups) {
         throw new Error(`could not find ${audit.id} audit for category ${categoryId}`);
       }
 
-      if (categoryId === 'accessibility' && !audit.group) {
+      const auditImpl = audits.find(a => a.implementation.meta.name === audit.id).implementation;
```

Comment:
maybe get rid of `auditIds` above and use `auditImpl` for both this and the check of `audit.id` above?

Reply:
done